Podfile.lock

```diff
@@ -1,20 +1,5 @@
 PODS:
   - 1PasswordExtension (1.8.5)
-  - AFNetworking (3.2.1):
```

Comment:
Any idea why the Podfile.lock changed so much without any changes to the Podfile?